database/migrations.php

```diff
@@ -63,8 +64,8 @@ function migrate()
     categoryId INT NOT NULL,
     startPrice DECIMAL(7,2) NOT NULL,
     reservePrice DECIMAL(7,2) CHECK (reservePrice > startPrice),
-    startDate TIMESTAMP(0) NOT NULL DEFAULT CURRENT_TIMESTAMP(0),
```

Comment:
We are going to have issues with this lol, I had to move it because in mac it was the other way around (got rejected if I put endDate before startDate... we might have to delete the CHECK and force this on the frontend if we keep having this issue.